service/crawlingContentService.js

```diff
@@ -13,8 +13,8 @@ const getCrawlingContentKeyword = async (req, res) => {
     const title = await page.$eval("title", (element) => element.textContent);
     let innerText = await page.$eval("body", (body) => body.innerText);
 
-    const hasTitleKeyword = title.includes(keyword);
-    let hasKeyword = innerText.includes(keyword);
+    const hasTitleKeyword = title.toUpperCase().includes(keyword.toUpperCase());
+    let hasKeyword = innerText.toUpperCase().includes(keyword.toUpperCase());
```

Comment:
포함여부를 검사할 때  영어를 모두 대문자화 시켜 더 일치하는 키워드를 많이 찾을 수 있는 좋은 작업인것 같습니다.
그런데 저는 toUpperCase() 메서드를 자주 실행시키기 보다 한 번만 실행 시킨 후 밑에선 실행 결과를 가져다 쓴다면 더욱 좋은 코드가 될 것 같다고 생각합니다.
```suggestion
    const upperCasedKeyword = keyword.toUpperCase(); 
    const hasTitleKeyword = title.toUpperCase().includes(upperCasedKeyword);
    let hasKeyword = innerText.toUpperCase().includes(upperCasedKeyword);
```

Reply:
저 또한 키워드를 대문자로 변환시키는 코드를 한 번만 실행 시킨 후 결과값을 밑에서 활용하는 것이 훨씬 코드의 가독성이 올라간다고 생각합니다. 수정 반영하겠습니다. 좋은 의견 감사합니다!